docs/nipoppy/configs.md

```diff
@@ -22,6 +22,10 @@ Nipoppy requires two global files for specifying local data/container paths and
       - Information about tabular data (`TABULAR`)
         - Version and path to the data dictionary (`data_dictionary`)
 
+!!! Note
```

Comment:
Would this be in a new page? Note that most of these (except `dicom_id` and `bids_id` are already in the docs: https://neurobagel.org/nipoppy/configs/#participant-manifest-manifestcsv)

Reply:
I think given the common confusion on these things, it would be good to have a "glossary" page. It's okay to have redundant explanation in the individual sections.